supertokens_python/logger.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import json
+import logging
+from datetime import datetime, timezone
+from os import getenv
+from typing import TextIO, Union
+
+from .constants import VERSION
+
+# Configure logger
+logger = logging.getLogger()
+log_level_str = getenv('LOG_LEVEL', "").lower()
+default_log_level = logging.INFO
+log_level_dict = {"debug": logging.DEBUG, "info": logging.INFO}
```

Comment:
We still seem to use multiple log levels, I think this should be more in line with the node PR (on or off)

Reply:
Removed. Thanks!